### DISCUSSION TITLE
[MultiSelect] customize group component

### DISCUSSION THREAD

**Discussion by ahetawal-p** [mantinedev/mantine#4733] [Q&A]
Hello,
We are trying to build custom `itemComponent` for the multiselect component, but we would also like to customize the grouping display, is there a way to do this. 
It would be basically a similar setup as `select all` functionality for a grouping.
Something like this:
![Screenshot 2023-09-06 at 2 43 33 PM](https://github.com/mantinedev/mantine/assets/8919529/1636213c-04ff-4009-978e-5594f3476e86)

**Comment by rtivital** [comment]
It is better to build a custom component with Input and Popover for this interface
